src/platform/datapath_raw_socket.c

```diff
@@ -408,11 +437,15 @@ CxPlatResolveRoute(
         Operation->Context = Context;
         Operation->Callback = Callback;
         Operation->PathId = PathId;
+        Route->State = RouteResolving;
         CxPlatDispatchLockAcquire(&Worker->Lock);
         CxPlatListInsertTail(&Worker->Operations, &Operation->WorkerLink);
         CxPlatDispatchLockRelease(&Worker->Lock);
         CxPlatEventSet(Worker->Ready);
         Status = ERROR_IO_PENDING;
+        CXPLAT_STATIC_ASSERT(
```

Comment:
You know, above we already set the Status variable to a QUIC_STATUS_XXX value (QUIC_STATUS_INVALID_STATE).

Reply:
I can remove it. I put it here mainly because it was a little special because you can't convert it using `HRESULT_FROM_WIN32`.

Reply:
Now, the code works so it's not needed anymore probably.